modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -0,0 +1,1200 @@
+			]
+		);
+
+		$this->add_control(
+			'icons_color',
+			[
+				'label'     => esc_html__( 'Color', 'elementor' ),
+				'type'      => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-link-in-bio__icon' => 'color: {{VALUE}}',
+				],
+			]
+		);
+
+		$this->add_control(
+			'icons_size',
+			[
+				'label'   => esc_html__( 'Size', 'elementor' ),
+				'type'    => Controls_Manager::SELECT,
+				'default' => 'small',
+				'options' => [
+					'small'  => esc_html__( 'Small', 'elementor' ),
+					'medium' => esc_html__( 'Medium', 'elementor' ),
+					'large'  => esc_html__( 'Large', 'elementor' ),
+				],
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .your-class' => 'border-style: {{VALUE}};',
+				// ],
+			]
+		);
+
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'cta_links_section_style',
+			[
+				'label' => esc_html__( 'CTA Links', 'elementor' ),
+				'tab'   => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'cta_links_type',
+			[
+				'label'   => esc_html__( 'Type', 'elementor' ),
+				'type'    => Controls_Manager::SELECT,
+				'default' => 'button',
+				'options' => [
+					'button' => esc_html__( 'Button', 'elementor' ),
+					'link'   => esc_html__( 'Link', 'elementor' ),
+				],
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .your-class' => 'border-style: {{VALUE}};',
+				// ],
+			]
+		);
+
+		$this->add_control(
+			'cta_links_text_color',
+			[
+				'label' => esc_html__( 'Text Color', 'elementor' ),
+				'type'  => Controls_Manager::COLOR,
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .your-class' => 'color: {{VALUE}}',
+				// ],
+			]
+		);
+
+		$this->add_group_control(
+			\Elementor\Group_Control_Typography::get_type(),
+			[
+				'name' => 'cta_links_typography',
+				// TODO: add class selector when markup is done
+				// 'selector' => '{{WRAPPER}} .your-class',
+			]
+		);
+
+		$this->add_control(
+			'cta_links_show_border',
+			[
+				'label'        => esc_html__( 'Border', 'elementor' ),
+				'type'         => Controls_Manager::SWITCHER,
+				'label_on'     => esc_html__( 'Yes', 'elementor' ),
+				'label_off'    => esc_html__( 'No', 'elementor' ),
+				'return_value' => 'yes',
+				'default'      => 'no',
+			]
+			// TODO: add class selector when markup is done
+			// 'selectors' => [
+			// 	'{{WRAPPER}} .elementor-tab-title' => 'border-width: {{SIZE}}{{UNIT}};',
+			// 	'{{WRAPPER}} .elementor-tab-content' => 'border-width: {{SIZE}}{{UNIT}};',
+			// ],
+		);
+
+		$this->add_control(
+			'cta_links_border_width',
+			[
+				'label'      => esc_html__( 'Border Width', 'elementor' ) . ' (px)',
+				'type'       => Controls_Manager::SLIDER,
+				'size_units' => [ 'px' ],
+				'range'      => [
+					'px' => [
+						'min'  => 0,
+						'max'  => 10,
+						'step' => 1,
+					],
```

Comment:
Maybe cache this in a variable/function somewhere? It keeps on repeating. It would be easy to change in one place. Just a suggestion.

Reply:
done!